usr/share/rear/skel/default/etc/scripts/unlock-opal-disks

```diff
@@ -201,6 +201,20 @@ while (( attempt < max_authentications )); do
         else
             display_message "$unlocked_device_count of $device_count disks unlocked, rebooting..."
         fi
+        if type -p efibootmgr &>/dev/null; then
```

Comment:
Just a shout question, why are you sending `type -p efibootmgr` to background using `&` ?

Reply:
@gozora 
`&>/dev/null` without a space between `&` and `>/dev/null`
is bash syntax shortcut to redirect both stdout and stderr to /dev/null

Reply:
SCNR:
"While the bash man page is technically 100% reliable,
its meaning by normal humans interpretation is not."
;-)

Reply:
That explains a lot ;-)
Thanks @jsmeix!

Reply:
Thanks for reviewing and resolving. Faster than I could notice. ;-)